tests/conftest.py

```diff
@@ -0,0 +1,115 @@
+import jinja2
+import logging
+from OpenSSL import crypto
+import os
+import pytest
+import subprocess
+import time
+
+
+pytest_plugins = [
+    "mock_fixtures",
+]
+
+
+def pytest_addoption(parser):
+    parser.addoption("--nginx", default="../nginx/objs/nginx")
```

Comment:
As discussed previously, the first two options need `default=...` replaced with `required=True`.

Reply:
Fixed.